README.md

```diff
@@ -114,10 +114,6 @@ It is recommended that you hook this in your favorite IDE (`Tools` > `File Watch
 
 ## Known Issues
 
```

Comment:
Hm, should we have some placeholder text to indicate that this is intentionally blank? Or just remove the section altogether until we know about some issues?

Reply:
Added placeholder text.